docs/deployment.md

```diff
@@ -119,3 +119,11 @@ To deploy a bot that includes multiple plugins, create a new app that has the pl
  }
 }
 ```
+
+## Error tracking
+
+Error tracking in production is of critical importance especially given that there is currently no recommended channel for letting the end user know that an error occurred. (We're working on a solution for end user error reporting [here](https://github.com/probot/friction/issues/1)). Probot comes bundled with [Raven](https://github.com/getsentry/raven-node) a client for the [Sentry](https://github.com/getsentry/sentry) exception tracking platform. To enable Sentry follow the following steps:
```

Comment:
> Error tracking in production is of critical importance especially given that there is currently no recommended channel for letting the end user know that an error occurred. (We're working on a solution for end user error reporting [here](https://github.com/probot/friction/issues/1)).

I feel like these could be removed and the docs would be just as helpful. These two sentences aren't wrong, they just aren't helpful.